Readme.md

```diff
@@ -19,11 +19,23 @@ oc {
     // Set either a token file or the token itself. The tokenFile takes precedence, if it exists
     tokenFile = File("service-account.token")
     // token = "secret-token"
+    // To login via plain text credentials
+    credentials {
+        username = "admin"
+        password = "admin"
```

Comment:
I suggest making at least the password parameter a closure. Then, you can either pass in a constant value, or you can do something like this (Groovy):
```
password = {
   return System.getenv("OC_PASSWORD")?.toCharArray()
}
```
That way, you can pass the password without risking committing a password accidentally.

Reply:
You can still do that inside of credentials action.

Reply:
Oh, I see. Would you mind adding an example to the Readme? :) I think that would help other people.

Reply:
Updated readme

Reply:
thx